src/main/java/org/apache/commons/text/names/HumanNameParser.java

```diff
@@ -98,8 +98,9 @@
  *
  * <p>This class is immutable.</p>
  */
-public final class HumanNameParser {
+public class HumanNameParser {
```

Comment:
Making HumanNameParser non-final can be discussed later in the mailing list or in another pull request (which gets sent to the mailing list anyway).

But for the scope of this pull request, I believe we must revert this change.